model/metrics/faas-metrics.yaml

```diff
@@ -0,0 +1,108 @@
+groups:
+  - id: metric.faas.invoke_duration
+    type: metric
+    metric_name: faas.invoke_duration
+    brief: "Measures the duration of the invocation"
+    instrument: histogram
+    unit: "ms"
+    attributes:
```

Comment:
nit:  attributes declaration/ref can be moved to separate 'common' attribute groups and then individual metrics can just extend this group.  

here's an example https://github.com/open-telemetry/semantic-conventions/blob/3b7bfcf811f814f01fd9a1435cffeee77b808e7d/semantic_conventions/metrics/messaging.yaml

Reply:
In this case, I refactored the attributes into a common file [faas-common.yaml](https://github.com/open-telemetry/semantic-conventions/pull/88/files#diff-87dba4dec93d12de40aae9915d20abae4d9ad87f8fa133bd88fd7442e598d4b6), but if I use `extends` it will bring all attributes from the group rendering the metrics with attributes they shouldn't have. Also, even if the group only had the exact set of attributes, using `extends` will automatically add newly introduced attributes to the metrics which I think is a bit dangerous.